app/controllers/PspDashboardController.scala

```diff
@@ -65,6 +66,7 @@ class PspDashboardController @Inject()(
             Ok(view(
               name = details.name,
               cards = service.getTiles(pspId, details),
+              Html(""), // Penalties tile does not, as yet, exist for PSP dashboard: only PSA dashboard.
```

Comment:
Is this meant to be here?

Reply:
yeah, for clarification, to be removed if/when we get a penalties tile for psp dashboard